projects/demo/src/modules/charts/line-days-chart/line-days-chart.component.ts

```diff
@@ -65,7 +65,9 @@ export class ExampleTuiLineDaysChartComponent {
         ],
     ];
 
-    value = this.valueVariants[0];
+    value: ReadonlyArray<[TuiDay, number]> = this.valueVariants[0] as ReadonlyArray<
+        [TuiDay, number]
```

Comment:
No need to typecast if you type `valueVariants`.

Reply:
fixed